### PR TITLE
[ML] PyTorch Sequence Classification (Sentiment Analysis) task

### PR DESCRIPTION
**Pull request by davidkyle**
Adds the Sequence Classification (Sentiment Analysis) task which given some input text returns a `positive_score` and `negative_score`. These values are softmax normalised.

```
POST _ml/trained_models/distilbert-finetuned-sst/deployment/_infer 
{
  "input": "The movie was awesome!!"
}

{
  "positive_score" : 0.9998062668825838,
  "negative_score" : 1.9373311741625532E-4
}

POST _ml/trained_models/distilbert-finetuned-sst/deployment/_infer 
{
  "input": "The cat was sick on the bed"
}
{
  "positive_score" : 0.0010437277318577885,
  "negative_score" : 0.9989562722681421
}
```

`InferenceResults` implement `ToXContentFragment` and so need to be wrapped in start and end objects in the API response. This change is required to be consistent but it means the NER and Fill Mask responses have changed to 
`{ entities: [ XXXX ]}` and `{ results: [ XXXX ]}`  rather than just returning the array.

### CODE REVIEW

**Review by dimitris-athanasiou — APPROVED** [review]
LGTM
